### PR TITLE
[TTS] remove CheckInstall

### PR DESCRIPTION
**Pull request by blisc**
# What does this PR do?

Please have one of the following:

* Removes CheckInstall from init.py

### CODE REVIEW

**Comment by Oktai15** [comment]
@blisc, should you update this file: https://github.com/NVIDIA/NeMo/blob/main/nemo/collections/tts/torch/readme.md?

**Comment by blisc** [comment]
> @blisc, should you update this file: https://github.com/NVIDIA/NeMo/blob/main/nemo/collections/tts/torch/readme.md?

Done

**Review by titu1994 — APPROVED** [review]
Thanks you @blisc  !

File: nemo/core/classes/common.py [hunk 1]
```diff
@@ -25,32 +25,22 @@
...
-    class SaveRestoreConnector(CheckInstall):
-        pass
-
-
 __all__ = ['Typing', 'FileIO', 'Model', 'Serialization', 'typecheck']
 
-
 _TYPECHECK_ENABLED = True
+# TODO @blisc: Remove _HAS_HYDRA
+_HAS_HYDRA = True
```
[the remark this answers is not in the document]
Comment (blisc) [reply nemo/core/classes/common.py:43]: I think it was used when I needed to import `core` but didn't install `hydra`. I'll remove it in another PR

[the remark this answers is not in the document]
Comment (titu1994) [reply nemo/core/classes/common.py:43]: Cool

**Pull request merged**